frontend/src/types.ts

```diff
@@ -2050,6 +2055,7 @@ export interface SessionRecordingPlayerProps {
     playerKey: string
     includeMeta?: boolean
     recordingStartTime?: string
+    matching?: MatchedRecording[]
```

Comment:
I think this name is a bit vague. Should it be `matchingEvents` and then the type be `MatchedRecordingEvents[]`?

Reply:
I wonder if it should be a dict then? Like `matching.events = []`. If we do want to expand the prop to other types, then we probably don't want those other types interlaced in the same lis.t